### PR TITLE
Feature/nm 18

### PR DESCRIPTION
**Pull request by amolcan**
Changed logic. Now we need to get all repositories for user, then iterate over repositories and get all pull requests for specific repository, then iterate over pull requests and get reviews for specific pull request and from reviews we filter only those which match our parameters.

### CODE REVIEW

File: controllers/comments.controller.js [hunk 1]
```diff
@@ -5,14 +5,15 @@ function getCommentsCountForUserName(req, res, next) {
     let userName = req.query.userName;
     let startDate = req.query.startDate;
     let endDate = req.query.endDate;
+    let pullRequestState = req.query.pullRequestState;
```
Comment (tjalc) [inline controllers/comments.controller.js]: why do we need add this parameter into request?

Comment (amolcan) [reply controllers/comments.controller.js]: To specify pull request state, so we can get comments count for all/open/closed pull requests. Default is open in GitHub API, so we need to set this parameter to all, if we want to get comments for all pull requests.

Comment (tjalc) [reply controllers/comments.controller.js]: I am not sure whether there is differentiation between comment on OPEN/CLOSED pull requests.
Even not sure whether we need to provide such parameter, but if we want I would definitely set default to all

Comment (amolcan) [reply controllers/comments.controller.js]: Changed, so we will get comments count for pull requests with state=all all the time.

**Comment by amolcan** [comment]
Test comment.

**Pull request merged**